### DISCUSSION TITLE
ScaduleController

### DISCUSSION THREAD

**Discussion by itafra-net** [IT-Afra/bakery-shop#65] [Q&A] [answered]
How do we complete the bread scadule controller?

**Comment by mohammadpirzadeh** [comment, marked as answer]
Hi  Afra .
The Method i Use : 
Step One :
Specify The "Route"   Sample :  ( [Route("api/[controller]")] )
Create a Class That Will Be Based On The Base Class
If We Use Dependency Injection, We Must Have Created An Item In The Services Section Of The Repository & Use It Here.
Otherwise, We Will Set An Example In The Constructor.
Finally, We Write The "Get" And "Post" Methods For It .

**Reply by itafra-net** [reply]
Ok Thnaks ,Mohmammad
